docs/getting_started.rst

```diff
@@ -20,32 +19,46 @@ LXDock should build very easily on Linux provided you have LXD available on your
 Prerequisite: install LXD
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-You may want to skip this section if you already have a working installation of LXD on your system.
+You may want to skip this section if you already have a working installation
+of LXD on your system.
 
-For Debian and Ubuntu, the following command will ensure that LXD is installed:
+For Debian or Ubuntu 16.04 and higher if prefer to stick with apt:
 
 .. code-block:: console
 
   $ sudo apt-get install lxd
 
+You can now also install LXD from Snap which works on Ubuntu 14.04 and higher.
+Since the PPA has been deprecated, this is now the easiest way to get
+the latest version of LXD on Ubuntu.
+
 .. note::
 
-  If you're using an old version of Ubuntu you should first add the LXD's apt repository and install
-  the ``lxd`` package as follows:
+  If you have already installed and configured LXD from apt earlier and
+  want to upgrade to the Snap version, you may need to purge LXD packages
+  first and reboot for the old network bridge to be removed. You can migrate
+  existing containers using ``lxd.migrate`` after or just start fresh.
 
   .. code-block:: console
 
-    $ sudo add-apt-repository -y ppa:ubuntu-lxc/lxd-stable
-    $ sudo apt-get update
-    $ sudo apt-get install lxd
+    $ sudo apt-get purge lxd lxd-client
+
+To install LXD from a Snap instead of apt:
+
+.. code-block:: console
+
+  $ sudo apt-get install snapd
+  $ sudo snap install lxd
+  $ sudo snap start lxd
 
-For Fedora, LXD is available through an experimental COPR repository. Unfortunately SELinux is not
-yet supported, therefore make sure it is disabled or set to permissive. Then run:
+For Fedora, LXD is available through an experimental COPR repository.
+Unfortunately SELinux is not yet supported, therefore make sure it is
+disabled or set to permissive. Then run:
 
 .. code-block:: console
 
-    # dnf copr enable ganto/lxd
```

Comment:
I'm not sure why this was using # and the others $ so I made it consistent.

Reply:
`#` implies a root shell. `$` implies a non-root shell. The original instruction is correct and the new ones are not.